__init__.py

```diff
@@ -1444,6 +1445,27 @@ def import_people(self, data, ignore_alias=False, raw_record_import=False):
             raw_record_import=raw_record_import,
         )
 
+    def import_from_amplitude(self, amplitude_api_key, amplitude_api_secret, start, end):
+        """Exports all data from an Amplitude project and imports it into Mixpanel
+        
+            :param amplitude_api_key: Your Amplitude API key
+            :param amplitude_api_secret: Your Amplitude API secret
+            :param start: Date and time in the format of YYYYMMDDTHH (e.g. '20150201T05')
+            :param end: Date and time in the format of YYYYMMDDTHH (e.g. '20150203T20')
+
+            :type amplitude_api_key: string
+            :type amplitude_api_secret: string
+            :type start: string
+            :type end: string
+        """
+        amplitude_export_url = f"https://amplitude.com/api/2/export?start={start}&end={end}"
+        credentials = f"{amplitude_api_key}:{amplitude_api_secret}"
+
+        extract_data_path = self._extract_amplitude_data(amplitude_export_url, credentials)
+        transform_data_path = self._transform_and_load_amplitude_data(extract_data_path)
```

Comment:
```suggestion
        self._transform_and_load_amplitude_data(extract_data_path)
```
doesn't look like this actually returns anything and it's not being used anyway